bin/triton-bootstrap

```diff
@@ -253,8 +260,9 @@ else
         echo -n '.'
 
         curl -s -i -f -u $COUCHBASE_USER:$COUCHBASE_PASS \
-            "http://$CLUSTERIP:8091/controller/addNode" \
-            -d "hostname=$(ip addr show eth0 | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')&user=admin&password=password"
+            -d "hostname=${IP_PRIVATE}&user=admin&password=password" \
```

Comment:
This is the moment that we're adding an unconfigured node to an already configured cluster. So we have to speak to the configured cluster using the user selected u/p, but tell it to add the new node using the factory default u/p.

It's worth adding that as a code comment, good call.